keyboards/cannonkeys/db60/config.h

```diff
@@ -19,11 +19,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xCA04
-#define PRODUCT_ID      0xDB60
 #define DEVICE_VER      0x0001
 
 #define MANUFACTURER CannonKeys
-#define PRODUCT DB60
+
+#if defined(KEYBOARD_cannonkeys_db60_j02)
+    #define PRODUCT_ID      0xDB60
+    #define PRODUCT DB60
+#elif defined(KEYBOARD_cannonkeys_db60_hotswap)
+    #define PRODUCT_ID      0xDB62
+    #define PRODUCT DB60 Hotswap
+#else
+    #define PRODUCT_ID      0xDB61
+    #define PRODUCT DB60v2
+#endif
```

Comment:
This should be done in separate config.h files.

Reply:
Is there any strong reason why?

Reply:
Just makes things simpler as all the revision-specific stuff is in one place instead of spread around the keyboard files.

The only place where we should be checking the keyboard name is the top-level `<keyboard>.h`, because it's necessary to include the header for the specific revision being compiled (see the Planck for example). The make logic can pick up everything else just fine on its own.

Reply:
Because this product uses different identifiers, a slightly different matrix, and can be easily confused by the name, wouldn't it make more sense to just make cannonkeys/bakeneko and then divide by hotswap or solder? This would not only make building firmware for newcomers easier to follow, but would also not break the existing J02 DB60 configuration. 
This is just my $0.02.

Reply:
These PCBs will potentially be used for other 60% boards with JST connectors, and I'd rather not make a new folder each time. I think it would be more confusing if other boards had to use the Bakeneko firmware. Though I understand where you are coming from.

The existing J-02 DB60 is not going to break - it's simply one of the revisions here.